Fix request.path inside with_request_url helper (#1463)
github/view_component: docs/CHANGELOG.md

## main

> * Fix `#with_request_url` (when used with query string) resulting in incorrect `request.path` and `request.fullpath`.

Reviewer: ```suggestion
* Fix bug where `#with_request_url`, when used with query string, set the incorrect `request.path` and `request.fullpath`.
```
